packages/react-components/react-accordion/stories/src/Accordion/AccordionCustomMotion.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { Accordion, AccordionHeader, AccordionItem, AccordionPanel } from '@fluentui/react-components';
+import { Collapse } from '@fluentui/react-motion-components-preview';
+import { createPresenceComponentVariant, motionTokens } from '@fluentui/react-components';
+import type { PresenceMotionSlotProps } from '@fluentui/react-motion';
+
+import description from './AccordionCustomMotion.stories.md';
+
+const CustomCollapseVariant = createPresenceComponentVariant(Collapse, {
+  enter: { duration: motionTokens.durationSlow, easing: motionTokens.curveEasyEaseMax },
+  exit: { duration: motionTokens.durationNormal, easing: motionTokens.curveEasyEaseMax },
+}) as PresenceMotionSlotProps;
+
+export const CustomMotion = () => (
+  <Accordion>
+    <AccordionItem value="1">
+      <AccordionHeader>Accordion Header 1</AccordionHeader>
+      <AccordionPanel collapseMotion={CustomCollapseVariant}>
+        <div>Accordion Panel with CustomCollapseVariant motion</div>
+      </AccordionPanel>
+    </AccordionItem>
+    <AccordionItem value="2">
+      <AccordionHeader>Accordion Header 2</AccordionHeader>
+      <AccordionPanel collapseMotion={CustomCollapseVariant}>
+        <div>Accordion Panel with CustomCollapseVariant motion</div>
+      </AccordionPanel>
+    </AccordionItem>
+  </Accordion>
+);
+
+CustomMotion.parameters = {
+  docs: {
+    description: {
+      story: description,
+    },
+  },
+};
```

Comment:
Let's do not add a story for a component itself. Instead we need to add general docs on customizing motion slots